### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by Ingramz**

### CODE REVIEW

File: lib/atom-io-client.coffee [hunk 1]
```diff
@@ -181,7 +181,7 @@ class AtomIoClient
 
       for key, children of files
         children.sort()
-        keep = children.pop()
+        children.pop() # keep
```
Comment (50Wliu) [inline lib/atom-io-client.coffee]: ?

File: lib/install-panel.coffee [hunk 2]
```diff
@@ -175,7 +171,7 @@ class InstallPanel extends ScrollView
         @searchMessage.hide()
         @searchErrors.append(new ErrorView(@packageManager, error))
 
-  showNoResultMessage: ->
+  showNoResultMessage: (query) ->
```
Comment (50Wliu) [inline lib/install-panel.coffee]: Nice!

File: lib/main.coffee [hunk 3]
```diff
@@ -60,7 +60,7 @@ module.exports =
       allPackages = values[1]
       if outdatedPackages.length > 0
         PackageUpdatesStatusView = require './package-updates-status-view'
-        packageUpdatesStatusView = new PackageUpdatesStatusView(statusBar, outdatedPackages)
+        new PackageUpdatesStatusView(statusBar, outdatedPackages)
```
Comment (50Wliu) [inline lib/main.coffee]: Is this and `PackageDetailView` (further below) still needed?

Comment (Ingramz) [inline lib/main.coffee]: Space pen views call `@initialize` of the extended class from constructor, which seems to be responsible for the magic of presenting itself. https://github.com/atom/settings-view/blob/c06ae51ac092d2a608d4cdff4677eeb2453e3e3d/lib/package-updates-status-view.coffee#L11-L14

File: lib/atom-io-client.coffee (hunk 1 above)
Comment (50Wliu) [inline lib/atom-io-client.coffee]: Also, this can be `children.sort().pop()` now right?

**Pull request merged**